README.md

```diff
@@ -100,7 +100,7 @@ json.set! :authorDetails, {...options} do
   json.set! :firstName, 'David'
 end
 
-or
+# or
```

Comment:
In code snippets like this, if non-ruby code is not commented it will mess up the syntax highligting:
![Screenshot 2025-01-02 at 10 23 31](https://github.com/user-attachments/assets/9e40fdaa-7703-4f1a-bffe-56107a36739f)